tests/test_other.py

```diff
@@ -10117,3 +10117,31 @@ def test_em_asm_side_module(self):
   def test_em_js_side_module(self):
     err = self.expect_fail([EMCC, '-sSIDE_MODULE', test_file('core/test_em_js.cpp')])
     self.assertContained('EM_JS is not supported in side modules', err)
+
+  # On Windows maximum command line length is 32767 characters. Create such a long build line by linking together
+  # several .o files to test that emcc internally uses response files properly.
```

Comment:
Perhaps mention here the places where to use response files internally?  IIUC that is when we run llvm-nm and wasm-ld(?).

Should we mark this test as windows only?  How about also marking it as slow (is it slow?)?